### DISCUSSION TITLE
store password used by budibase outside budibase

### DISCUSSION THREAD

**Discussion by sohail-hussain** [Budibase/budibase#5329] [Help]
Running budibase in a corporate network where we store password externally in a Hasicorp vault

is this an available feature?

**Comment by melohagan** [comment]
Hi @sohail-hussain 

There is an API for Hashicorp vault: https://www.vaultproject.io/api-docs

Then you can the budibase REST API connector: https://docs.budibase.com/docs/rest

A word of caution is that the X-Vault-Token header may be included in app exports, so you probably want to make sure they stay within the company.
